src/mumble/Database.cpp

```diff
@@ -238,6 +241,26 @@ void Database::setLocalIgnored(const QString &hash, bool ignored) {
 	execQueryAndLogFailure(query);
 }
 
+bool Database::isLocalIgnoredTTS(const QString &hash) {
+	QSqlQuery query(db);
+
+	query.prepare(QLatin1String("SELECT `hash` FROM `ignored_tts` WHERE `hash` = ?"));
+	query.addBindValue(hash);
+	execQueryAndLogFailure(query);
+	return query.next();
+}
+
+void Database::setLocalIgnoredTTS(const QString &hash, bool ignoredTTS) {
+	QSqlQuery query(db);
+
+	if (ignoredTTS)
+		query.prepare(QLatin1String("INSERT INTO `ignored_tts` (`hash`) VALUES (?)"));
+	else
+		query.prepare(QLatin1String("DELETE FROM `ignored_tts` WHERE `hash` = ?"));
```

Comment:
Does this query fail, if `hash` is not in the table? (Failing as in "crash the application")

Reply:
I'll look into that.

It's done the same way as local muting and such. So if this code is problematic, these need to change too.

Reply:
No, it does not crash. I set the value, manually deleted it from sqlite and the unset it. Nothing happened.